lib/collection/src/shards/local_shard/clock_map.rs

```diff
@@ -15,6 +15,9 @@ use crate::shards::shard::PeerId;
 #[serde(from = "ClockMapHelper", into = "ClockMapHelper")]
 pub struct ClockMap {
     clocks: HashMap<Key, Clock>,
+    /// Whether this clock map has changed since the last time it was persisted.
+    #[serde(skip, default)]
```

Comment:
default should be `true`?

Reply:
How so?

It's either from disk or a new one. In both cases I'd consider it unchanged, because if we load it in the same way we'd get the exact same state.

Reply:
If it is new, it should be `changed=true`, otherwise how we save it? I think. No?

Reply:
We could do it to be on the safe side. But the above mentioned function makes it unnecessary in the current implementation.

Reply:
If we are Ok, with not saving default, then let's consider it resolved.